.github/workflows/docker-image.yml

```diff
@@ -19,13 +19,23 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
       - name: Login to DockerHub
-        uses: docker/login-action@v1 
+        uses: docker/login-action@v2.0.0
         with:
           username: ${{ secrets.DOCKER_USER }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2.0.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ github.token }}
       - name: Build and push the docker image
         uses: docker/build-push-action@v2
         with:
           platforms: linux/amd64,linux/arm64,linux/arm/v7
           push: true
-          tags: timothyye/godns:latest,timothyye/godns:${{steps.tag.outputs.tag}}
+          tags: |
+            timothyye/godns:latest
```

Comment:
If dockerhub and github has the same user name and package name, this could be replaced by ${{github.repository}} as well.